pkg/models/execution.go

```diff
@@ -138,6 +144,8 @@ type Execution struct {
 	CreateTime int64 `json:"CreateTime"`
 	// ModifyTime is the time the execution was last updated.
 	ModifyTime int64 `json:"ModifyTime"`
+
+	Evaluation *Evaluation `json:"Evaluation"`
```

Comment:
o we care about passing the evaluation here or jump make the details available at the execution level

Reply:
Done